src/android/Zoom.java

```diff
@@ -168,43 +181,33 @@ private void initialize(String appKey, String appSecret, CallbackContext callbac
             return;
         }
 
+        ZoomSDKInitParams params = new ZoomSDKInitParams();
+        params.appKey = appKey;
+        params.appSecret = appSecret;
+        params.domain = this.WEB_DOMAIN;
+        params.enableLog = true;
 
-        try {
```

Comment:
Now the initialisation should be done using ZoomSDKInitParams & ZoomSDKInitializeListener as per the new documentation.